dask/array/tests/test_routines.py

```diff
@@ -1896,6 +1896,149 @@ def test_where_incorrect_args():
             assert "either both or neither of x and y should be given" in str(e)
 
 
+def _test_array_equal_parametrizations():
```

Comment:
Could you use a proper fixture here?

Reply:
I'm not sure I understand, could you explain in a bit more detail what you would like to see, please? I read through all the documentation on `pytest.mark.parametrize` and fixtures, but I cannot find how using a fixture would facilitate parametrizing a test. Or do you mean to drop `pytest.mark.parametrize` and write a single test that loops over all the test cases? That would prevent running the tests in parallel though.

As a side note: this test setup was copied from [numpy/_core/tests/test_numeric.py](https://github.com/numpy/numpy/blob/80acab201eb5445af925552468983073616284dc/numpy/_core/tests/test_numeric.py#L1979-L2124), so it might be easier for future maintenance to keep modifications to a minimum.